documentation/asciidoc/computers/camera/raspicam.adoc

```diff
@@ -1,3 +1,46 @@
+== Raspicam applications
+
+[WARNING]
+====
+Raspberry Pi has transitioned from a legacy camera software stack based on proprietary Broadcom GPU code to an open-source stack based on `libcamera`. As such, the _Raspicam_ stack is now deprecated.  Raspberry Pi OS images from _Bullseye_ onwards will contain *only* the `libcamera`-based stack. Raspberry Pi OS images up to and including _Buster_ will contain the legacy _Raspicam_ stack, though the `libcamera` stack and applications can be installed xref:camera_software.adoc#libcamera-and-libcamera-apps-packages[using _apt_], or built by following the xref:camera_software.adoc#building-libcamera-and-libcamera-apps[normal build instructions].
```

Comment:
Perhaps remove the "will" from "Raspberry Pi OS images from _Bullseye_ onwards will contain" ?

Reply:
Done